### PR TITLE
[DROOLS-6276] Rule relationship analysis graph : Brush-up for experim…

### PR DESCRIPTION
**Pull request by tkobayas**
…ental release

- Add example usage test

**JIRA**: 

https://issues.redhat.com/browse/DROOLS-6301

**referenced Pull Requests**: 

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1630

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by tkobayas** [comment]
Jenkins retest this

**Comment by tkobayas** [comment]
Failure in CDB:
~~~
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project jbpm-flow-builder: Compilation failure
[ERROR] /home/jenkins/workspace/KIE/master/daily-build/daily-build-pipeline-master/jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/xml/XmlProcessReader.java:[57,50] cannot find symbol
[ERROR]   symbol: method getParents()
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project jbpm-flow-builder: Compilation failure
/home/jenkins/workspace/KIE/master/daily-build/daily-build-pipeline-master/jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/xml/XmlProcessReader.java:[57,50] cannot find symbol
  symbol: method getParents()
~~~
was caused by the commit https://github.com/kiegroup/drools/commit/295bdb69e83d60cbdfd06977fc0cfa275ae0f0ba#diff-97bf99e18a0fd7f03fa96683930ea47cd6ddbb11f084cd85d33f012a2722726bL624

File: pom.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
     <module>drools-alphanetwork-compiler</module>
     <module>drools-engine</module>
     <module>drools-engine-classic</module>
-<!--    <module>drools-impact-analysis</module>-->
+    <module>drools-impact-analysis</module>
```
Comment (tkobayas) [inline pom.xml:88]: Now we build the module!

**Comment by tkobayas** [comment]
@mariofusco @hellowdan Please review, thanks!

**Review by mariofusco — APPROVED** [review]

**Review by hellowdan — APPROVED** [review]

**Pull request merged**